board/raspberrypi/bluetooth-remote.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
This file must be moved the BluetoothRemoteContro, plugin folder and must be named after the vendor. 
And when copying the file (in CMakeList.txt see as an example in the RemoteControl plugin) rename it to the any prefered name